src/screens/settings/SettingsReaderScreen/Settings/CustomCSSSettings.tsx

```diff
@@ -26,7 +26,10 @@ const CustomCSSSettings = () => {
       </View>
       <View style={styles.customCSSContainer}>
         <Text numberOfLines={3} style={[{ color: theme.onSurface }]}>
-          {customCSS || 'Example: body {margin: 10px;}'}
+          {customCSS ||
+            getString(
+              'moreScreen.settingsScreen.readerSettings.customCSSPlaceholder',
+            )}
```

Comment:
I think only `Example` is enough. we cant translate code tho

Reply:
Then we make code as const?

Reply:
sure, translating the code is kinda cursing